### PR TITLE
adding_schedule_library_correct_quantity

### PR DESCRIPTION
**Pull request by jposada202020**
Solves #76

### CODE REVIEW

**Comment by jposada202020** [comment]
@lesamouraipourpre could you review, I just change the number of libraries and include the Schedule library to the list. Thanks

File: circuitpython_community_library_list.md [hunk 1]
```diff
@@ -44,6 +44,7 @@ Here is a listing of current CircuitPython Community Libraries. There are 43 lib
 * [CircuitPython HMAC](https://github.com/jimbobbennett/CircuitPython_HMAC) ([PyPi](https://pypi.org/project/circuitpython-hmac)) \([Docs](https://circuitpython.readthedocs.io/projects/hmac/en/latest/))
 * [CircuitPython MorseCode](https://github.com/jposada202020/CircuitPython_MorseCode.git) \([Docs](https://circuitpython-morsecode.readthedocs.io/en/latest/))
 * [CircuitPython Parse](https://github.com/jimbobbennett/CircuitPython_Parse) ([PyPi](https://pypi.org/project/circuitpython-parse)) \([Docs](https://circuitpython.readthedocs.io/projects/parse/en/latest/))
+* [CircuitPython_Schedule](https://github.com/cognitivegears/CircuitPython_Schedule.git) ([PyPi](https://pypi.org/project/circuitpython-parse)) \([Docs](https://circuitpython-schedule.readthedocs.io/en/latest/))
```
Comment (lesamouraipourpre) [inline circuitpython_community_library_list.md:47]: The PyPi link should be
https://pypi.org/project/circuitpython-schedule

**Review by lesamouraipourpre — CHANGES_REQUESTED** [review]
Other than the PyPI link this looks grand.

File: circuitpython_community_library_list.md (hunk 1 above)
Comment (jposada202020) [reply circuitpython_community_library_list.md]: Thanks. This is corrected in aa6f5f7

**Comment by jposada202020** [comment]
Thanks for reviewing :)

**Comment by jposada202020** [comment]
@lesamouraipourpre let me know if this is ok.. Thanks :)

**Review by lesamouraipourpre — APPROVED** [review]
Looks good to me.

**Pull request merged**